playwright.config.ts

```diff
@@ -0,0 +1,79 @@
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// import dotenv from 'dotenv';
+// dotenv.config({ path: path.resolve(__dirname, '.env') });
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
```

Comment:
It looks like some of this can be removed and/or simplified, perhaps it can be setup more like the config in bods-integrated-data?

Reply:
I think this setup needs to be different as this is for UI testing but I will remove the comments